platform-configs/include/default.robot

```diff
@@ -143,6 +143,7 @@ ${CPU_TESTS_SUPPORT}=                               ${FALSE}
 ${HYPER_THREADING_SUPPORT}=                         ${FALSE}
 ${INTEL_HYBRID_ARCH_SUPPORT}=                       ${FALSE}
 ${RESET_TO_DEFAULTS_SUPPORT}=                       ${FALSE}
+${L2_CACHE_SUPPORT}=                                ${FALSE}
```

Comment:
It was supposed to be TRUE?

Reply:
removed in: 2cde25bac3dd60533fbc565a0ca6010772a8d1b6